starlite/openapi/controller.py

```diff
@@ -125,56 +183,83 @@ def root(self, request: Request) -> str:
                 A [Request][starlite.connection.Request] instance.
 
         Returns:
-            A rendered html string.
+            resoponse: With the rendered site defined in root_schema_site.
```

Comment:
`resoponse` 😉 , you should be more careful with copy/paste.

Please link the response references here as above.